dark/index.html

```diff
@@ -7,7 +7,8 @@
   <meta charset="utf-8" />
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
-  <meta name="description" content="Hi, I am Yoshitha Rathnayake. I am from Kandy, Sri Lanka. I am 14 years old. I am studing in St. Anthony's College, Kandy" />
+  <meta name="description"
+    content="Hi, I am Yoshitha Rathnayake. I am from Kandy, Sri Lanka. I am 14 years old. I am studing in St. Anthony's College, Kandy" />

```

Comment:
We don't need this one element in two lines, it happens after formatting the code undo them @Rishav1707

Reply:
Undo these @Rishav1707